web-components/src/components/cards/ReviewCard/ReviewCard.Photo.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  CardMedia,
+  CardMediaProps,
+  useTheme,
+  useMediaQuery,
+} from '@mui/material';
+
+import Card from '../Card';
+
+const Photo: React.FC<CardMediaProps> = ({ src }) => {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+
+  return (
+    <Card
+      sx={{
+        mt: 9,
+        mb: 2,
+        height: [216, 293],
+      }}
+    >
+      <CardMedia component="img" src={src} height={isMobile ? 216 : 293} />
```

Comment:
```suggestion
      <CardMedia component="img" src={src} sx={{height:{xs: 216, sm:293}}} />
```

Reply:
TY. For some reason this didn't work at first for me. Maybe it was a caching issue.